app/src/main/res/values/strings.xml

```diff
@@ -100,8 +100,8 @@
     <string name="create">Create</string>
     <string name="delete">Delete</string>
     <string name="edit">Edit</string>
-    <string name="folder">Folder</string>
-    <string name="folder_name">Folder Name</string>
+    <string name="folder">Folder*</string>
```

Comment:
This has an unintended effect of showing the * in the dialog pop up when you click the + to add a folder or post